File: python/at_least_two_greater_elements.py
# https://www.geeksforgeeks.org/problems/at-least-two-greater-elements4625/1

class Solution:
    def findElements(self,arr):
       arr.sort()
       arr.pop()
       arr.pop()
        
       return arr

#{ 
 # Driver Code Starts
#Initial Template for Python 3


def main():

    T = int(input())

    while (T > 0):

        arr = [int(x) for x in input().strip().split()]
        ob = Solution()
        print(*ob.findElements(arr))

        T -= 1


if __name__ == "__main__":
    main()

# } Driver Code Ends

File: javascript/at_least_two_greater_elements.js
// https://www.geeksforgeeks.org/problems/at-least-two-greater-elements4625/1

class Solution {
  findElements(arr) {
    let res = [...arr];
    res.sort((a, b) => a - b); // sort in ascending order

    res.pop(); // deletion of last element is o(1) time
    res.pop();

    return res;
  }
}
